server/embed/serve.go

```diff
@@ -95,6 +95,8 @@ func (sctx *serveCtx) serve(
 	tlsinfo *transport.TLSInfo,
 	handler http.Handler,
 	errHandler func(error),
+	grpcDial func(ctx context.Context) (*grpc.ClientConn, error),
```

Comment:
Please document why we do need grpc function. 
It's very counter intuitive when I looked at the call... and signature...

Reply:
Maybe call it `grpcDialForRestGatewayBackends`